docs/docs/examples/vault.md

```diff
@@ -0,0 +1,290 @@
+effectively restricting communication to attested Contrast deployments.
+The Coordinator’s transit secrets engine API authorizes requests based on the `workloadSecretID`,
+which is embedded in a certificate extension and must match the target endpoint.
+As previously noted, updating the `workloadSecretID` after initializing the LUKS device will make it inaccessible,
+due to a mismatch in the derived encryption key.
+Therefore, it is critical to ensure that the `workloadSecretID` is correctly aligned with the intended endpoint
+specified in Vault’s sealing configuration before the first `contrast set` is executed.
+
+
+## Verifying the deployment as a user
+
+In different scenarios, users of an app may want to verify its security and identity before sharing data, for example, before connecting to the database.
+With Contrast, a user only needs a single remote-attestation step to verify the deployment - regardless of the size or scale of the deployment.
+Contrast is designed such that, by verifying the Coordinator, the user transitively verifies those systems the Coordinator has already verified or will verify in the future.
+Successful verification of the Coordinator means that the user can be sure that the given manifest will be enforced.
+
+### Verifying the Coordinator
+
+A user can verify the Contrast deployment using the verify
+command:
+
+```sh
+contrast verify -c "${coordinator}:1313" -m manifest.json
+```
+
+The CLI will verify the Coordinator via remote attestation using the reference values from a given manifest. This manifest needs
+to be communicated out of band to everyone wanting to verify the deployment, as the `verify` command checks
+if the currently active manifest at the Coordinator matches the manifest given to the CLI. If the command succeeds,
+the Coordinator deployment was successfully verified to be running in the expected Confidential
+Computing environment with the expected code version. The Coordinator will then return its
+configuration over the established TLS channel. The CLI will store this information, namely the root
+certificate of the mesh (`mesh-ca.pem`) and the history of manifests, into the `verify/` directory.
+In addition, the policies referenced in the manifest history are also written into the same directory.
+
+### Auditing the manifest history and artifacts
+
+In the next step, the Coordinator configuration that was written by the `verify` command needs to be audited.
+A user of the application should inspect the manifest and the referenced policies. They could delegate
+this task to an entity they trust.
+
+
+### Connecting to the application
+
+Other confidential containers can securely connect to the Vault server via the
+[Service Mesh](../components/service-mesh.md).
+Any verified deployment with a valid Service Mesh cerrtificate is trusted by the Vault server application and thus can execute Vault-related operations
```

Comment:
There's punctuation missing at the end of this line, and I think the meaning of `trust` could be clearer here: it's rather that the mesh certificate is a requirement to even talk to the Vault endpoint, but by itself it does not grant any permissions.